packages/optimizer/lib/transformers/SeparateKeyframes.js

```diff
@@ -60,8 +68,19 @@ class SeparateKeyframes {
       },
     };
 
+    const isInvalidKeyframe = (keyframe) => {
+      return keyframe.keyframes.some((k) => {
```

Comment:
A good old nested for loop might be more performant here.

Reply:
Updated